pkg/minikube/service/service.go

```diff
@@ -332,10 +332,7 @@ func CreateSecret(namespace, name string, dataValues map[string]string, labels m
 		return &retry.RetriableError{Err: err}
 	}
 	secrets := client.Secrets(namespace)
-	secret, err := secrets.Get(name, meta.GetOptions{})
-	if err != nil {
-		return &retry.RetriableError{Err: err}
-	}
+	secret, _ := secrets.Get(name, meta.GetOptions{})
```

Comment:
At a minimum, this should log or display an error rather than hide it entirely. I'm not exactly sure which is appropriate here, but naively glog.Errorf?

Reply:
Thanks, logging makes sense. I feel a glog.Infof would be most appropriate as I don't think it should be an error when CreateSecrets is unable to get a secret it is going to create.